src/OneDriveSdk/Models/Generated/Item.cs

```diff
@@ -56,7 +56,7 @@ public partial class Item
         public DateTimeOffset? CreatedDateTime { get; set; }
     
         /// <summary>
-        /// Gets or sets c tag.
+        /// Gets or sets ctag.
```

Comment:
This is a generated file so this edit will be overwritten at some point. Same comment for the below file.